webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -438,6 +442,7 @@ class ValuesResponderV1 extends ResponderV1 {
                                             maybeValuePermissions = permissionsFromDefaults
                                         ).toString()
 
+                                        // append the SPARQL that was generated to create standoff links to the other insert statements
```

Comment:
@benjamingeer Can this comment be removed? This was my fix to the problem that `standoffLinkInsertSparql` was never used, but now you added it to `allInsertSparql` (line 466). Should the comment be moved there?